proposals/proposal-template.md

```diff
@@ -0,0 +1,98 @@
+# JSON Schema Proposal: <TODO>
+
+## Abstract
+
+<!--
+Fill in the specification(s) that will change.  If adding a keyword, which vocabulary
+will contain it?
+
+For example
+
+    This document proposes a change to the JSON Schema Core specification and
+    Applicator vocabulary by adding the `propertyDependencies` keyword.
+
+-->
+
+This document proposes a change to the ??? specification by adding ???.
+
+## Status
+
+**Current Status**: PROPOSAL
+
+<!-- TODO: We should have a short standard blurb outlining the stages involved in a
+feature making its way to stable status. -->
+
+<!-- TODO: Link to a document that describes the proposal => stable process in
+detail. -->
+
+## Note to Readers
+
+<!--
+It would be a good idea to isolate the issues that discuss the feature by
+updating the link below to include a query.
+-->
+
+The issues list for this proposal can be found at
+<https://github.com/json-schema-org/json-schema-spec/issues>.
+
+For additional information, see <https://json-schema.org/>.
+
+To provide feedback, use this issue tracker or any of the communication methods
+listed on the homepage.
+
+## Table of Contents
+
+## Conventions and Terminology
+
+All conventions and terms used and defined by the JSON Schema Core specification
+also apply to this document.
+
+## Overview
+
+### Problem Statement
+
+<!-- What problem exists that needs solving? -->
+
+### Solution
+
+<!-- What is the solution?  Include examples of use. -->
+
+### Alternatives
+
+<!-- What other options have been considered? (summary, not detailed) -->
```

Comment:
Per https://github.com/json-schema-org/json-schema-spec/issues/1443#issuecomment-2099427543, maybe this should go.  This would be important for an ADR, but it's not necessarily something that the public needs to see.

Reply:
I'm still not entirely sure where you want to draw the line between the two documents, but I agree that a discussion of the alternatives isn't necessary in this document except maybe as an appendix.

Reply:
Please see my https://github.com/json-schema-org/json-schema-spec/issues/1443#issuecomment-2108834298